docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md

```diff
@@ -92,8 +92,14 @@ To connect the board:
     !!! tip
         To show available subcommands, use the **Tab** key.
 
-0. Optionally, change the default credentials by following the instructions in the program.
+0. Check your default credentials by following the instructions in the program.
     ![Anjay configuration](images/anjay_config.png "Anjay configuration")
+
+    !!! note
+        If your default credentials are different than device credentials from Coiote IoT Device Management, please change them with `anjay config set possible_option` command.
+        <br/>
+        ![Anjay set configuration](images/anjay_config_set.PNG "Anjay set configuration"){:style="float: left;margin-right: 1177px;margin-top: 7px;"}
+
```

Comment:
Is there a possibility to insert some space into `Note` section below the image? The bottom of the image is at the same height as the bottom of the `Note` section.